src/main/java/com/codeborne/selenide/commands/SetSelected.java

```diff
@@ -13,9 +13,23 @@
   public WebElement execute(SelenideElement proxy, WebElementSource locator, Object[] args) {
     boolean selected = (Boolean) args[0];
     WebElement element = locator.getWebElement();
-    if (element.isSelected() ^ selected) {
-      if (element.getAttribute("readonly") != null)
-        throw new InvalidStateException("Cannot change value of readonly element");
+    if (!element.isDisplayed()) {
+      throw new InvalidStateException("Cannot change invisible element");
+    }
+    String tag = element.getTagName();
+    if (!tag.equals("options")) {
```

Comment:
why `options`? There is no such html tag.
And you haven't added any tests with `options`.

Reply:
right you are, thanx!